src/Root.tsx

```diff
@@ -0,0 +1,25 @@
+import {
+  HashRouter as Router,
+  Routes,
+  Route,
+  Navigate,
+} from 'react-router-dom';
+import { App } from './App';
+import { HomePage } from './components/HomePage';
+import { PeoplePage } from './components/PeoplePage';
+import { PageNotFound } from './components/PageNotFound';
+
+export const Root = () => (
+  <Router>
+    <Routes>
+      <Route path="/" element={<App />}>
+        <Route index element={<HomePage />} />
+        <Route path="home" element={<Navigate to="/" replace />} />
+        <Route path="people" element={<PeoplePage />}>
+          <Route path=":slugId" element={<PeoplePage />} />
```

Comment:
It seems like the nested route ':slugId' is pointing to the same component 'PeoplePage'. If this is intentional, ensure that 'PeoplePage' can handle both the list and detail views. Otherwise, consider creating a separate component for the detail view.